src/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
I thinks we can remove all commented key to have something more clean. What do you thinks ?

Reply:
Good suggestion. I did that.

Reply:
👍